ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -1059,4 +1097,53 @@ end
 - [ ] `To seed the database with initial data.`
 - [ ] `To test the database connection.`
 
-[reference](https://guides.rubyonrails.org/v3.2/migrations.html#:~:text=Active%20Record%20tracks%20which%20migrations,the%20structure%20of%20your%20database.)
+[Reference](https://guides.rubyonrails.org/v3.2/migrations.html#:~:text=Active%20Record%20tracks%20which%20migrations,the%20structure%20of%20your%20database.)
+
+#### Q79. What is the execution result of the following Ruby code?
+
+```ruby
+class A
+  def self.get_self
+    self.class
+  end
+
+  def get_self
+    self
+  end
+end
+
+p "#{A.get_self.class} #{A.new.get_self.class}"
+```
+
+- [ ] `Class Class`
+- [ ] `A A`
+- [ ] `A Class`
+- [x] `Class A`
+
+[Reference](https://www.ignacio.al/self-class-ruby.html)
+
+#### Q80. Given the following model class and view , which code snippet should you use to fetch the correct results, and avoid the N+1 query issue?
```

Comment:
New question concerning rails N+1 queries. It's not the first one, but it's one I haven't seen everywhere.